core/src/ast/modules.rs

```diff
@@ -310,15 +310,19 @@ impl Module {
             }
             Item::Macro(mac) => {
                 if let Some(i) = &mac.ident {
-                    let macro_rules_attr = mac.attrs.iter().find_map(move |a| {
-                        if a.path()
-                            == &syn::parse_str::<syn::Path>("diplomat::macro_rules").unwrap()
-                        {
-                            Some(true)
-                        } else {
-                            None
-                        }
-                    }).unwrap_or(false);
+                    let macro_rules_attr = mac
```

Comment:
nit: I htink this can be `mac.attrs...find(|a| a.path == ... )` without any of the option or unwrap_or stuff